examples/nofsverity/build

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+set -eux
+
+os="${1:-fedora}"
+cd "${0%/*}"
+
+FIX_VERITY=1 ../common/check-config
+
+case "${os}" in
+    arch)
+        containerfile='Containerfile.arch'
+        features='--features=pre-6.15'
+        ;;
+    *)
+        echo "*** unknown variant ${os}"
+        false
+        ;;
+esac
+
+# https://github.com/containers/buildah/issues/5656
+PODMAN_BUILD="podman build --no-cache"
+
+cargo build --release "${features}"
+
+cp ../../target/release/cfsctl .
+cp ../../target/release/composefs-setup-root extra/usr/lib/dracut/modules.d/37composefs/
+CFSCTL='./cfsctl --repo tmp/sysroot/composefs'
+
+rm -rf tmp
+mkdir -p tmp/sysroot/composefs
+
+${PODMAN_BUILD} \
+    --iidfile=tmp/base.iid \
+    --target=base \
+    -f "${containerfile}" \
+    .
+
+BASE_ID="$(sed s/sha256:// tmp/base.iid)"
+${CFSCTL} oci pull containers-storage:"${BASE_ID}"
+BASE_IMAGE_FSVERITY="$(${CFSCTL} oci compute-id --bootable "${BASE_ID}")"
+
+${PODMAN_BUILD} \
+    --iidfile=tmp/final.iid \
+    --build-context=base="container-image://${BASE_ID}" \
+    --build-arg=COMPOSEFS_FSVERITY="?${BASE_IMAGE_FSVERITY}" \
```

Comment:
It seems like just about the only thing changed is that you add the `?` here and the `-sr` below when building the image, ya?  I think that's probably not worth a whole separate scenario....

Reply:
It might also be interesting to try building an image without fs-verity but with the `composefs=` cmdline specified without the `?`, ie: to make sure that it fails as expected.